src/components/PaymentLog.vue

```diff
@@ -618,9 +618,17 @@ export default {
             return item.name === "renewal";
           })?.id)
         : (renewal = "");
-      let orderType = this.orderTypes.find((item) => {
+        let orderType = '';
+        if(this.isAltaraPay){
+           orderType = this.orderTypes.find((item) => {
```

Comment:
why not handle it something like this?
 `orderType = this.orderTypes.find((item) =>  (item.name === this.isAltaraPay ? "Altara Pay": "Altara Credit"));`